_inc/lib/core-api/wpcom-endpoints/publicize-connection-test-results.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+require_once dirname( __FILE__ ) . '/publicize-connections.php';
+
+/**
+ * Publicize: List Connection Test Result Data
+ *
+ * All the same data as the Publicize Connections Endpoint, plus test results.
+ *
+ * @since 6.8
+ */
+class WPCOM_REST_API_V2_Endpoint_List_Publicize_Connection_Test_Results extends WPCOM_REST_API_V2_Endpoint_List_Publicize_Connections {
+	public function __construct() {
+		$this->namespace = 'wpcom/v2';
+		$this->rest_base = 'publicize/connection-test-results';
+
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Called automatically on `rest_api_init()`.
+	 */
+	public function register_routes() {
+		register_rest_route( $this->namespace, '/' . $this->rest_base, array(
+			array(
+				'methods'             => WP_REST_Server::READABLE,
+				'callback'            => array( $this, 'get_items' ),
+				'permission_callback' => array( $this, 'get_items_permission_check' ),
+			),
+			'schema' => array( $this, 'get_public_item_schema' ),
+		) );
+	}
+
+	/**
+	 * Adds the test results properties to the Connection schema.
+	 */
+	public function get_item_schema() {
+		$schema = array(
+			'$schema' => 'http://json-schema.org/draft-04/schema#',
+			'title' => 'jetpack-publicize-connection-test-results',
+			'type' => 'object',
+			'properties' => $this->get_connection_schema_properties() + array(
+				'test_success' => array(
+					'description' => __( 'Did the Publicize Connection test pass?', 'jetpack' ),
```

Comment:
It makes sense that we capitalize `Publicize`, but I think we should make an audit in the whole PR and lowercase any `Connection` or `Service` that occurs in the middle of a phrase.

Reply:
I wanted to make it clear it was a Publicize Connection, etc. and not an HTTP connection, Jetpack connection, etc. I'm happy to change it, though.

Reply:
Thanks, given the context I'm good with it as-is 👍